packages/csharp/ArmoniK.Api.Common/Options/ComputePlane.cs

```diff
@@ -38,6 +38,16 @@ public class ComputePlane
   /// </summary>
   public const string SettingSection = nameof(ComputePlane);
 
+  /// <summary>
+  ///   Path to the section containing the worker channel values in configuration object
+  /// </summary>
+  public const string WorkerChannelSection = "WorkerChannel";
+
+  /// <summary>
+  ///  Path to the section containing the agent channel values in configuration object
+  /// </summary>
+  public const string AgentChannelSection = "AgentChannel";
```

Comment:
Is it possible to put a `nameof` here ?
```suggestion
  public const string AgentChannelSection = nameof(AgentChannel);
```